docs/pages/blog/first-look-at-joy.md

```diff
@@ -0,0 +1,133 @@
+
+## Why work on it?
+
+You might intuitively associate MUI with Material UI, which despite the indeed potentially confusing names, are separate things.
+Material UI is MUI's React implementation of Google's Material Design.
+
+Over the years Material UI has established itself as the go-to library for quickly breathing life into products, mostly thanks to its design, customizability, and documentation.
+However, the components do come with the default 2018 Google look and feel that is no longer as popular as it was.
+And, as we've reconfirmed with [our latest developer survey](/blog/2021-developer-survey-results/#what-are-your-most-important-criteria-for-choosing-a-ui-library/), the design aspect of UI libraries is one of the most important ones whenever evaluating which to choose.
+
+## Why not just build a different theme?
+
+One of the main objectives with this new product is to break away from Material Design.
+A different theme would still require you and your team to have at least some sense of how Material Design is structured, and how that is expressed in Material UI. We wanted to provide an alternative to that. With Joy, we want you to be able to start with a fresh design by default.
+
+We also had many ideas for improving the experience of developing design systems, so a separate package is an opportunity to prototype and test some of these new features without running the risk of compromising the experience of the thousands of teams that use Material UI today.
+
+Material UI and Joy UI will still share many conventions and features.
+This will not only decrease the learning curve for adopting Joy, but also because we think of them both representing the ideal developer experience for developing design systems.
+So if at any time Joy UI has a feature that Material UI doesn't, it will catch-up at some point, and vice versa.
+
+## What are the new things we're introducing in Joy UI?
+
+Aside from a fresher look and feel for your upcoming project, here are a few features we're planning for the alpha version of Joy UI:
+
+### Global variants
+
+Instead of defining variants at the component level, Joy UI defines variants at the global level.
+This sounds bold at first, but the more we experiment the more it starts to make sense.
+[Global variants](/joy-ui/core-features/global-variant/) mostly affect three CSS properties: color, background, and border.
+
+What this means is that you'll be able to switch between the same set of variants in any component, more easily maintaining consistency across your app.
+The four variants available in the components are `plain`, `outlined`, `soft`, and `solid`.
```

Comment:
```suggestion
The four variants available in the components are `solid`, `soft, `outlined` and `plain`.
```
So that we match the order of which they are shown in the demo :) 
![image](https://user-images.githubusercontent.com/4512430/171835544-d108cabb-360d-481a-aff9-852c3b7969a2.png)